### ISSUE TITLE
Double normalization

### ISSUE THREAD

**Issue by Yiwen233** [ts-kim/RevIN#17]
Thanks for your work. I read the source code of Informer with RevIN, and I found that the data is normalized by standard normalization, and then sent to the RevIn layer. So there are actually two normalization steps by default. So is that the way to enhance performance and get the result in your paper?

**Comment by PlanckChang** [comment]
I also found this issue. Do you have any new ideas recently?

**Comment by khchul** [comment]
Yeah I also think this is a big issue that the authors should answer. Because RevIN is casually used in many recent Time Series papers, the double normalization problem appears very frequently. I'm also wondering if the result of the paper could change if the scale variable in the dataloader is set to false.

**Comment by Yiwen233** [comment]
> I also found this issue. Do you have any new ideas recently?

Not yet

**Comment by SamKnightGit** [comment]
I would also be interested in the Author's weighing in on this. Table 3 in the paper seems to suggest that RevIN replaces standard normalization techniques but if standardization is necessary for the RevIN layer to reproduce the results, this should probably be made explicit. I plan on replicating a subset of the results with NBEATS on the ETTh1 dataset, with and without pre-standardization of the input data, I'll update here with my results.

**Comment by ikhsansdqq** [comment]
Hi, I've currently researching RevIN usage on iTransformer, LSTM, and RNN, I only apply once for the scaling either it's minmax or revin, the results can be seen as follow:

![Image](https://github.com/user-attachments/assets/70bda3ab-d18a-4ac8-8cf0-ec4212237736)

**Comment by g42alaxy** [comment]
> Hi, I've currently researching RevIN usage on iTransformer, LSTM, and RNN, I only apply once for the scaling either it's minmax or revin, the results can be seen as follow:
> 
> ![Image](https://github.com/user-attachments/assets/70bda3ab-d18a-4ac8-8cf0-ec4212237736)

Would you be open to share this reproduction code?

**Comment by ikhsansdqq** [comment]
Hi, thanks for the enthusiasm! However, our current code is not yet available to the public to GitHub.
